includes/data-stores/class-wc-admin-reports-data-store.php

```diff
@@ -159,9 +158,9 @@ protected function fill_in_missing_intervals( $db_intervals, $datetime_start, $d
 	 * @param stdClass $data           Data from whose intervals the records get removed.
 	 * @param int      $page_no        Offset requested by the user.
 	 * @param int      $items_per_page Number of records requested by the user.
-	 * @param int      $db_interval_count
-	 * @param int      $expected_interval_count
-	 * @param string   $order_by
+	 * @param int      $db_interval_count Database interval count.
+	 * @param int      $expected_interval_count Expect DB interval count.
```

Comment:
Since the data in db can be sparse, this is not an expected db interval count, but expected interval count on the output.